src/v2/guide/index.md

```diff
@@ -331,18 +331,18 @@ var app7 = new Vue({
   el: '#app-7',
   data: {
     groceryList: [
-      { text: 'Vegetables' },
-      { text: 'Cheese' },
-      { text: 'Whatever else humans are supposed to eat' }
+      { text: 'Légumes' },
+      { text: 'Fromage' },
+      { text: 'Tout ce que les humains sont supposés manger' }
     ]
   }
 })
 </script>
 {% endraw %}
 
-This is just a contrived example, but we have managed to separate our app into two smaller units, and the child is reasonably well-decoupled from the parent via the props interface. We can now further improve our `<todo-item>` component with more complex template and logic without affecting the parent app.
+Ceci n'est qu'un exemple grossier, mais nous avons séparé notre application en deux plus petites unités, et chaque enfant est convenablement découplé de son parent via l'utilisation des props. Nous pouvons maintenant améliorer notre `<todo-item>` avec un template et une logique plus conséquente sans affecter l'application parente.
```

Comment:
"et chaque enfant" -> "et l'enfant" 
(the child c'est le singulier, j'imagine que ça fait référence au "composant enfant" plutôt qu'à ses instances)